modules/platforms/cpp/thin-client/src/impl/message.h

```diff
@@ -658,7 +658,7 @@ namespace ignite
                     writer.WriteInt8(concurrency);
                     writer.WriteInt8(isolation);
                     writer.WriteInt64(timeout);
-                    writer.WriteString(label.Get());
+                    writer.WriteString(label.IsValid() ? label.Get() : "");
```

Comment:
According to specs, you should write null, not empty string.

Reply:
got it ! thanks !